mobile/src/framework/notifications/notifications-base.package.ts

```diff
@@ -16,10 +19,12 @@ class BaseNotifications implements Notification {
         });
     }
 
-    public showSuccess(value: string): void {
+    public showMessage(settings: ShowMessageArguments): void {
+        const { type, text, title } = settings;
         this.notify.show({
-            type: 'success',
-            text1: value,
+            type,
+            text1: title,
+            text2: text,
```

Comment:
rename back to showSuccess 
also values for title or text can be optional, are they optional?

Reply:
I have refactored success and error message for more consistent. Title is required, because we don't need empty notifications, text is optional